entries/u/united.com.json

```diff
@@ -2,12 +2,20 @@
   "United": {
     "domain": "united.com",
     "url": "https://www.united.com/",
-    "contact": {
-      "facebook": "United",
-      "twitter": "united"
-    },
-    "categories": [
+    "tfa": [
+      "email",
+      "sms",
+      "call",
+      "totp",
+      "custom-software"
+    ],
+     "custom-software": [
+      "United App"
+    ],
+    "documentation":"https://www.united.com/en/us/fly/mileageplus/protect-your-account.html",
+    "notes": "Requires E-Mail address to establish 2FA, TOTP only available through United app",
```

Comment:
Don't you need an email to sign up for a United account? As mentioned before, the United app does not qualify as TOTP because it is a proprietary software method.